puddles/Backplane/src/depth_sensor.c

```diff
@@ -323,7 +323,7 @@ static void depth_adc_queue_reads(int num_reads, void (*callback)(void)) {
  */
 static int64_t depth_read_alarm_callback(__unused alarm_id_t id, __unused void *user_data) {
     if (depth_read_running) {
-        LOG_ERROR("Depth new transaction started with one still in progress");
+        // LOG_ERROR("Depth new transaction started with one still in progress");
```

Comment:
Re-enable depth failure logging?

Reply:
This one was really spammy, and is very easy to hide the root cause. Just raising the error should be fine